src/helpers/helpers.go

```diff
@@ -69,6 +79,26 @@ func JobKubernetesToMetronome(job *v1beta1.CronJob) *MetronomeJob {
 	return ret
 }
 
+// Convert Kubernetes Job to Metronome job run format
+func JobKubernetesToMetronome(job *batchv1.Job) *MetronomeJobRun {
+	ret := &MetronomeJobRun{}
+	ret.Id = fmt.Sprintf("%s.%s", job.ObjectMeta.Namespace, job.ObjectMeta.Name)
+	ret.JobId = fmt.Sprintf("%s.%s", job.ObjectMeta.Namespace, job.ObjectMeta.OwnerReferences[0].Name)
+	ret.CreatedAt = job.ObjectMeta.CreationTimestamp.String()
+	if job.Status.StartTime == nil {
+		ret.Status = "STARTING"
+	} else if job.Status.CompletionTime == nil {
+		ret.Status = "RUNNING"
+	} else {
```

Comment:
optional: look for job.Status.Failed and set ret.Status to FAILED

Reply:
It seems that `job.Status.Failed` is an `int32` vs. a `bool`, so I'm not entirely sure how it's supposed to work.